src/test/java/lihua/tasks/EventTest.java

```diff
@@ -0,0 +1,4 @@
+package lihua.tasks;
+
+public class EventTest {
```

Comment:
Are you supposed to add content to this test class file, because this class does not serve a purpose now as it is empty